cross/fuse/patch/002-somepatch.patch

```diff
@@ -0,0 +1,24 @@
+--- util/Makefile.in.org    2015-08-19 10:56:51.834947550 +0200
```

Comment:
Great looking patch :P The name could use some work though...how about `cross-compile.patch`...something a bit more descriptive.